NorthstarDedicatedTest/bansystem.cpp

```diff
@@ -20,7 +21,25 @@ void ServerBanSystem::OpenBanlist()
 	{
 		std::string line;
 		while (std::getline(enabledModsStream, line))
-			m_vBannedUids.push_back(strtoull(line.c_str(), nullptr, 10));
+		{
+			// ignore line if first char is # or line is empty
+			if (line == ""  || line.front() == BANLIST_COMMENT_CHAR)
```

Comment:
```suggestion
			if (line == "" || line.front() == BANLIST_COMMENT_CHAR)
```

This is where format check failed btw